integration-tests/openapi-java/pom.xml

```diff
@@ -154,10 +171,28 @@
                         <artifactId>maven-failsafe-plugin</artifactId>
                         <executions>
                             <execution>
+                                <id>profile-openapi-v2</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <test.profile>openapi.v2</test.profile>
+                                    </systemPropertyVariables>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>profile-openapi-v3</id>
                                 <goals>
                                     <goal>integration-test</goal>
                                     <goal>verify</goal>
                                 </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <test.profile>openapi.v3</test.profile>
+                                    </systemPropertyVariables>
+                                </configuration>
                             </execution>
```

Comment:
This will cost some work to make it work on the platform. I guess there is no way to have multiple OpenApi versions in a single app (so that we can we can avoid this Maven config)?

Is there no need to have the same for surefire?

Reply:
> Is there no need to have the same for surefire?

That's done via `@TestProfile` that works only in JVM mode?

Reply:
> > Is there no need to have the same for surefire?
> 
> That's done via `@TestProfile` that works only in JVM mode?

Correct, the additional bits added to the `pom.xml` were to support native mode. So shouldn't be an issue for platform as native is not tested there (AFAIK).

Reply:
Their CI does not, but IMO, we should take care to have native runable anyway. I sometimes do run native tests there. OK, so it needs porting https://github.com/apache/camel-quarkus/issues/3075